### ISSUE TITLE
Should users be able to vote on topics?

### ISSUE THREAD

**Issue by Luse** [LinSangM/lean-coffee#1]
In lean coffee topics can be voted on, should this also be the case for this project? :)

**Comment by LinSangM** [comment]
Yes... it will add to the the planning there soon. But we are not sure yet how to implement it, add a new phase for it ? or just let it editing phase and voting phase working at the same time
